Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM rust:1.69-bullseye as rust-build-stage
+RUN apt-get update && apt-get upgrade -y \
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
```

Comment:
why is this needed (in a multi-stage docker file)?

Reply:
You're right, this isn't necessary in this case, since it's a multi-stage build. I'll take it out!